### ISSUE TITLE
Busybox is broken

### ISSUE THREAD

**Issue by 1q23lyc45** [tiann/KernelSU#2254]
### Please check before submitting an issue

- [X] I have searched the issues and haven't found anything relevant
- [X] I will upload bugreport file in KernelSU Manager - Settings - Report log
- [X] I know how to reproduce the issue which may not be specific to my device

### Describe the bug

```
root@localhost:~# /data/adb/ksu/bin/busybox udhcpc6 -i wlan0
stack corruption detected (-fstack-protector)
Aborted
```

### To Reproduce

1. Install Termux 0.118.1 from F-Droid or Github
2. Run command in termux: sudo /data/adb/ksu/bin/busybox udhcpc6 -i wlan0

### Device info

- Device: It's not relevant.
- OS Version: It's not relevant.
- KernelSU Version: 11986
- Kernel Version: It's not relevant.

### Additional context

Also I would suggest to remove /userspace/ksud/bin and use CI to auto-build it instead. Otherwise, it is prone to problems.

**Comment by 1q23lyc45** [comment]
fixed in PR #2383
Not sure why I submitted PR #2252 more than a month ago but you didn't merge. Today, @aviraxp re-submitted the same PR in #2383, but you merged it.
I wonder what the reason is.

**Comment by 1q23lyc45** [comment]
fixed in PR #2383
Not sure why I submitted PR #2273 more than a month ago but you didn't merge. Today, @aviraxp re-submitted the same PR in #2383, but you merged it.
I wonder what the reason is.

**Comment by tiann** [comment]
> fixed in PR [#2383](https://github.com/tiann/KernelSU/pull/2383) Not sure why I submitted PR [#2273](https://github.com/tiann/KernelSU/pull/2273) more than a month ago but you didn't merge. Today, [@aviraxp](https://github.com/aviraxp) re-submitted the same PR in [#2383](https://github.com/tiann/KernelSU/pull/2383), but you merged it. I wonder what the reason is.

I'm very sorry, I can't remember at all why this PR was closed; I recall that someone once submitted a PR that also updated files, but the content it updated was empty, and I may have confused the two. I apologize again for my foolish behavior!

**Comment by 1q23lyc45** [comment]
> > fixed in PR [#2383](https://github.com/tiann/KernelSU/pull/2383) Not sure why I submitted PR [#2273](https://github.com/tiann/KernelSU/pull/2273) more than a month ago but you didn't merge. Today, [@aviraxp](https://github.com/aviraxp) re-submitted the same PR in [#2383](https://github.com/tiann/KernelSU/pull/2383), but you merged it. I wonder what the reason is.
> 
> I'm very sorry, I can't remember at all why this PR was closed; I recall that someone once submitted a PR that also updated files, but the content it updated was empty, and I may have confused the two. I apologize again for my foolish behavior!

That's all right. It doesn't a matter. Don't blame yourself.
